src/compile/selection/transforms/project.ts

```diff
@@ -56,6 +55,31 @@ const project: TransformCompiler = {
       return {[range]: signals[sg] = sg};
     };
 
+    // If no explicit encodings are specified, set some defaults.
+    // If an initial value is set, try to infer projections for more concise specification.
```

Comment:
> for more concise specification.

Is it just more concise?  I thought we need this to make it init work at all?

Reply:
Depends how you parse the sentence. But I remove the clause to avoid confusion and further wordsmithing.